app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixPriceRangeBottomSheet.kt

```diff
@@ -0,0 +1,173 @@
+                                  text = "0$",
+                                  color = colorScheme.onBackground,
+                                  style = TextStyle(fontWeight = FontWeight.SemiBold))
+                            }
+
+                        Row(modifier = Modifier.weight(0.70f).testTag("priceRangeSlider")) {
+                          QuickFixPriceRange(
+                              modifier = Modifier.fillMaxWidth().padding(horizontal = 8.dp),
+                              rangeColor = colorScheme.primary,
+                              backColor = colorScheme.secondaryContainer,
+                              barHeight = 8.dp,
+                              circleRadius = 8.dp,
+                              cornerRadius = CornerRadius(10f, 10f),
+                              minValue = 0,
+                              maxValue = 3000,
+                              progress1InitialValue = 500,
+                              progress2InitialValue = 2500,
+                              tooltipSpacing = 5.dp,
+                              tooltipWidth = 50.dp * widthRatio.value,
+                              tooltipHeight = 30.dp * heightRatio.value,
+                              tooltipTriangleSize = 5.dp,
+                              onProgressChanged = { value1, value2 ->
+                                range_0 = value1
+                                range_1 = value2
+                              })
+                        }
+
+                        Row(
+                            modifier = Modifier.weight(0.15f).testTag("rightPriceText"),
+                            horizontalArrangement = Arrangement.Start) {
+                              Text(
+                                  text = "3000$",
+                                  color = colorScheme.onBackground,
+                                  style = TextStyle(fontWeight = FontWeight.SemiBold))
+                            }
+                      }
+
+                  Spacer(modifier = Modifier.height(verticalSpacing * 1.5f))
+
+                  Button(
+                      onClick = {
+                        onApplyClick(range_0, range_1)
+                        Log.d("hey", range_0.toString())
+                        Log.d("hey", range_1.toString())
+                        onDismissRequest()
+                      },
+                      modifier =
+                          Modifier.fillMaxWidth()
+                              .padding(horizontal = buttonPaddingHorizontal)
+                              .testTag("applyButton"),
+                      colors =
+                          androidx.compose.material3.ButtonDefaults.buttonColors(
+                              containerColor = colorScheme.primary,
+                              contentColor = colorScheme.onPrimary)) {
+                        Text("Apply")
+                      }
+                }
+          }
+        }
+  }
+}
+
+@Preview(showBackground = true)
```

Comment:
Preview is for testing purposes only, but since the app is not in production yet we can keep it for now

you may resolve this